src/core/builtins/builtins_files.ml

```diff
@@ -120,10 +124,16 @@ let _ =
     ]
     Lang.unit_t
     (fun p ->
+      let parents = List.assoc "parents" p |> Lang.to_bool in
       let perms = List.assoc "perms" p |> Lang.to_int in
       let dir = List.assoc "" p |> Lang.to_string in
       try
-        Unix.mkdir dir perms;
+        let rec recmkdir dir =
+          if not (Sys.file_exists dir) then (
+            recmkdir (Filename.dirname dir);
+            Unix.mkdir dir perms)
+        in
+        if parents then recmkdir dir else Unix.mkdir dir perms;
         Lang.unit
       with _ -> Lang.unit)
```

Comment:
Should we raise a runtime error instead?

Reply:
Let's keep the simple behavior for now.